modules/software-extensibility/pages/extensions-sdk.adoc

```diff
@@ -0,0 +1,28 @@
+= Extensions Source Development Kits(SDK)
+:description: Several Source Development Kits are provided so that you can extend your automation project
+
+{description}
+
+
+[.card-section]
+== The provided SDKs
+
+[.card.card-index]
+--
+xref:connector-archetype.adoc[[.card-title]#Connector Maven Archetype# [.card-body.card-content-overflow]#pass:q[Create custom Connectors]#]
```

Comment:
You have to reference the module since this page is in an other module. 
Please use the preview to check your contribution, you would have notified it quickly.

```suggestion
xref:process:connector-archetype.adoc[[.card-title]#Connector Maven Archetype# [.card-body.card-content-overflow]#pass:q[Create custom Connectors]#]
```